Pyspark/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM bitnami/spark:latest
+
+# Adding PYTHONPATH and workdirectory
+ENV PYTHONPATH="${PYTHONPATH}:/Pyspark"
+ENV PYARROW_IGNORE_TIMEZONE=1
+WORKDIR /Pyspark
```

Comment:
For future please do not name project folders similar to library names like folede 'Pysaprk' and lib `pyspark`.
1. It can be misleading for other people to distinguish 'who is who' when looking into imports
2. Common pattern for naming python modules and packages are starting from lowercase